utils/vectorscan/patches/0001-CMakeLists.txt-dirty-hack.patch

```diff
@@ -0,0 +1,19 @@
+From 5ce0e124f04d7b69f3257b57a71873d14a00c9e4 Mon Sep 17 00:00:00 2001
+Date: Tue, 23 May 2023 06:52:19 -0400
+Subject: [PATCH] CMakeLists.txt: dirty hack
```

Comment:
This sounds good... 🤦 

1. Why this patch should be accepted?
2. Who is the author of this patch?
3. Did you try to reach upstream to take care of this?

So many questions...

Reply:
1. The patch should not be part of the final PR, it is only there to force the build system to pass the correct values for my RPi4 for testing purposes only.
2. I am the author but again, this patch will not be included in the final PR
3. I did not yet contact upstream... I am wanting someone more familiar with OpenWRT's build system to look at the `CMakeLists.txt` provided by upstream and let me know if the OpenWRT build system can work around this via passing some variables or some other bit of code.  What is your recommendation, @BKPepe for this?